pkg/cli/sync.go

```diff
@@ -158,5 +160,28 @@ func (c *SyncCommand) Run(ctx context.Context, args []string) error {
 	// TODO(#72): create reader, writer base on cmd flags.
 	// TODO(#71): create group and user mapping proto and textproto parser.
 
+	sm, dm, err := client.NewBidirectionalNewOneToManyGroupMapper(c.source, c.destination, c.groupMappingConfig)
```

Comment:
should we get rid of the extra `New` in the name? i.e `NewBidirectionalOneToManyGroupMapper`

Reply:
Done